scripts/release.sh

```diff
@@ -412,9 +344,12 @@ else
     nix flake check
 
     report_progress "Checks run successfully."
-    confirm_proceed "  -- Please add the release notes to RELEASES.md if not already done. Save but don't commit. Then press 'y'."
+    confirm_proceed "  -- Please add the release notes to RELEASES.md if not already done and save. Then press 'y'."
 
-    report_progress "Pusing the release branch..."
+    git add RELEASES.md
```

Comment:
This has been a long-standing bug for the release script unrelated to this PR (the missing `git add`), but each time I fix it and forgot to commit it to the master branch (given the branch juggling involved when doing a release), so I'm just taking the occasion.